07_StagingAndMultipleQueues/main.cpp

```diff
@@ -23,6 +22,13 @@ class StagingAndMultipleQueuesApp final : public examples::BasicMultiQueueApplic
 		using device_base_t = examples::BasicMultiQueueApplication;
 		using asset_base_t = examples::MonoAssetManagerAndBuiltinResourceApplication;
 
+		static constexpr uint32_t IMAGE_CNT = 3u;
```

Comment:
you can actually deduce this variable from `decltype(imagesToLoad)` since this will compile
```cpp
		static constexpr auto imagesToLoad = std::array<std::string_view>({
			"../app_resources/test0.png",
			"../app_resources/test1.png",
			"../app_resources/test2.png"
		});
```